src/organisation/models/update-pba-numbers.model.ts

```diff
@@ -0,0 +1,42 @@
+export class UpdatePbaNumbers {
+  public existingPbaNumbers: string[] = [];
+  public pendingAddPbaNumbers: string[] = [];
+  public pendingRemovePbaNumbers: string[] = [];
+
+  constructor(existingPbaNumbers: string[]) {
+    this.existingPbaNumbers = existingPbaNumbers;
+  }
+
+  get hasDisplayPbaNumbers(): boolean {
+    return this.displayPbaNumbers.length > 0;
+  }
+
+  get displayPbaNumbers(): string[] {
+    return this.existingPbaNumbers.filter(pba => this.pendingRemovePbaNumbers.indexOf(pba) === -1);
+  }
+
+  /**
+   * Current PBA Numbers contain existing and pending additions, minus pending removals
+   */
+  get currentPbaNumbers(): string[] {
+    const currentPbas = this.existingPbaNumbers
+      .concat(this.pendingAddPbaNumbers)
+      .filter(pba => this.pendingRemovePbaNumbers.indexOf(pba) === -1);
+
+    return currentPbas;
+  }
+
+  public addPbaNumberToPendingRemove(pba: string): void {
+    const alreadyInPendingRemove = this.pendingRemovePbaNumbers.indexOf(pba) != -1;
```

Comment:
`!==`?

Reply:
I just like to assign to variables so that if statements read more like English, i.e. "if already in pending remove". But I am happy to change.

Reply:
made change as requested